bench/src/test/scala/Benchmarks.scala

```diff
@@ -74,11 +76,24 @@ object BenchTests extends OnlineRegressionReport {
   def setup =
     performance of "dotty" in {
       measure.method("stdlib") in {
-        using(Gen.unit("test")) curve "stdlib" in { r => stdLib }
+        // maybe scalac curve later
+        using(Gen.unit("test")) curve "dotty" in { r => stdLib }
```

Comment:
@DarkDimius The `curve` is to support multiple curves in a chart. Yes, I should keep it the same for `stdlib` and `dotty-src` for backward data compatibility.